File: 3/A_12_425.java
// Болібрук Влад
public class A_12_425 {
	public static void main(String[] args) {
		float s = -10;
		float t = 20;
		float result = calculateRes(1.2f, s) + calculateRes(t, s) - calculateRes(2 * s - 1, s * t);


		System.out.println("task is A_12_425");
		System.out.printf("result is %f\n", result);
	}

	public static float calculateRes(float a, float b) {
		return (a * a + b * b) / (a * a + 2 * a * b + 3 * b * b + 4);
	}
}

File: S1/1/A_12_425.c
// Болібрук Влад
#include <stdio.h>
#include <stdlib.h>
#include <math.h>

float calculateRes(float a,float b);

int main()
{
    float s = -10;
    float t = 20;
    float result = calculateRes(1.2, s)+calculateRes(t, s) - calculateRes(2*s-1, s*t);


    printf("task is A_12_425\n");
    printf("result is %f\n", result);

    return 0;
}
float calculateRes(float a,float b){
    return ((a*a)+(b*b))/((a*a)+(2*a*b)+(3*b*b)+4);
}
